upgrading/upgrade-running-applications/init_background.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
-
-tar xzf create-daml-app.tar.gz
+cd /tmp
```

Comment:
did you figure out why you have to do this and if we could simply set the path instead?